README.md

```diff
@@ -1 +1,10 @@
-# angular-filter-track-by-lab
\ No newline at end of file
+# Using track by inside ng-repeat and $filter in Controller
+
+## Objectives
+
+- Refactor our ng-repeat to use a Controller filter
+- Integrate "track by" clause to enhance DOM render change performance
+
+## Instructions
+
```

Comment:
Here, add a brief explanation of the what the starter code is and the context is for the lab. Explain that we have a contact list that we want to be able to filter through. Then continue on to the instructions below.